doc/source/report/ultradian.rst

```diff
@@ -1,92 +1,66 @@
 .. _ultradian:
 
-Ultradian Analysis
-==================
+Ultradian and Circadian Analysis
+================================
 
 Statement of problem
 --------------------
 
-Ultradian rhythm is widely observed in mammalian behavioral patterns.
-Ultradian analysis aims to find the time-specific patterns in behavioral
-records, without specifying the length of cycle in advance (but need to be
-within 1 hour to 1 day). Typical ultradian period for rats includes 4, 12 and
-24 hours. For example, we expect rats to be inactive in the nighttime.
-Ingestions and movements mostly happened in the daytime. It would be
-informative to study the ultradian cycle of the behavior of mouse and we need
-to answer the following questions for this study:
-
-What is the variable of interest for the periodic patterns? 
-
-- Summary of activity: Food and water ingestion, distance traveled , movement
-  intensity, AS probability.
-
-- Spatial variable: Spatially discrete the data to cells each with its primary
-  functions such as food cell, water cell, etc. Examine ultradian cycle of the
-  spatial probability densities of the occupancy time in each cells.
-
-How to subset the data?
-
-- Basic subset: 16 strains.
-
-- Strains may not be the primary influence for the variation of ultradian
-  rhythms. We may look into the cycle for each mouse and detect the most
-  important factors influencing the ultradian rhythms.
-
-How to choose the frequency or period ?
-
-- The Lomb-Scargle (LS) periodogram spectral analysis technique, a widely used
-  tool in period detection and frequency analysis.
-
-What is the connection with other subprojects?
-
-- Ultradian rhythms could be treated as one feature for clustering the 16
-  strains. We may also subset the data using the results of the cluster and
-  analysis the rhythm similarities and differences across clusters.
+Ultradian and circadian rhythm are widely observed in mammalian
+behavioral patterns. The ultradian analysis aims to
+find the time-specific patterns in behavioral
+records, without specifying the length of the cycle in advance (but need to be
+within 1 hour to 1 day). The typical ultradian period includes 8 and 12 hours.
+The circadian rhythm refers to a roughly 24-hour cycle.
+For example, we expect the rats to be inactive in the nighttime.
+And ingestions and movements mostly happened in the daytime.
+
+The variables of interests are the summary statistics of mouse activity
+including food (F) and water (W) ingestion, distance traveled (D), Active
+State probability (AS), movement inside the home base (M_IS) and
+movement outside the home base (M_AS). We may also consider spatial variables,
+for example, to spatially discrete the data to cells each with its primary
+functions such as food cell, water cell, and examine
+ultradian cycle of the spatial probability densities
+of the occupancy time in each cell.
```

Comment:
"We may also consider spatial variables. For example,
we can spatially discretize the data into cells containing the primary
functions, like a food cell or a water cell, and examine the
ultradian cycle of the spatial probability densities
of the occupancy time in each cell."